.github/workflows/ci.yml

```diff
@@ -285,7 +293,7 @@ jobs:
         working-directory: ./tee-worker
         shell: bash
         run: |
-          for d in . identity/enclave-runtime bitacross/enclave-runtime; do
+          for d in . identity identity/enclave-runtime; do
```

Comment:
Hmm why this change?

Reply:
previously `tee-check` checks `enclave-runtime` from both `identity` and `bitacross`. But didn't check other files within `identity` and `bitacross`. 
I changed it to two jobs, each of them check files both within `enclave-runtime` and outside.

Reply:
I think it does.

`for d in . identity/enclave-runtime bitacross/enclave-runtime; do`

does the workspace-wide check as they belong to the same workspace

Reply:
Yes. You are right they are in the same workspace. I reverted the changes.